youth-leading-the-way.html

```diff
@@ -0,0 +1,536 @@
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z"
+                /></svg
+            ></span>
+          </button>
+          <ol class="panel">
+            <li>
+              Participating in training and information sessions from Helping
+              Hands
+            </li>
+            <li>
+              Any 1-on-1 meetings with Helping Hands: These can be as far back
+              from June 12th
+            </li>
+            <li>
+              Other volunteer activities you have done from June 12th, 2023 and
+              onwards Just let us know and we can reach out to them to confirm
+              your completed hours.
+            </li>
+            <li>
+              Any applications, or surveys completed for Helping Hands and/or
+              CSC can count towards an hour.
+            </li>
+          </ol>
+
+          <button class="accordion">
+            What kinds of volunteer opportunities are there?
+            <span>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="16"
+                height="16"
+                fill="currentColor"
+                class="bi bi-caret-down"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z"
+                /></svg
+            ></span>
+          </button>
+          <ol class="panel">
+            <li>
+              Depending on what is available at the time, volunteer
+              opportunities may look very different. They can range from:
+            </li>
+            <li>
+              Helping to tutor online with an international non-profit
+              organization
+            </li>
+            <li>
+              Volunteering at your local library, hospital, community center
+            </li>
+            <li>
+              Mentoring a youth, volunteering as a coach or referee, volunteer
+              reading buddy, helping older adults with technology, teen advisory
+              group,...etc
+            </li>
+            <li>
+              There may be multiple placements needed for you to fulfill enough
+              hours
+            </li>
+            <li>
+              You can be volunteering in the front or from behind the scenes!
+            </li>
+          </ol>
+        </div>
+      </div>
+
+      <!-- last section -->
+      <div class="interested">
+        <button class="landing-btn-last">
+          If you are an organization that is interested in hosting some
+          volunteers for this program you are welcome to email
+          community@helpinghandsapp.com for more information.
+        </button>
```

Comment:
Don't use the button tag if it doesn't do anything when clicked. If you want the aesthetic of a button, but it doesn't do anything, use div.